pandas/tests/indexes/period/test_indexing.py

```diff
@@ -3,10 +3,11 @@
 import pytest
 
 import numpy as np
+from numpy import testing as ntm
```

Comment:
NO, we never use numpy.testing.! remove!

use ``tm.assert_numpy_array_equal``.